tests/check/check_signed_video_auth.c

```diff
@@ -2257,7 +2259,7 @@ START_TEST(golden_sei_principle)
   // Final validation is OK and all received NAL Units, but the last one, are validated.
   signed_video_accumulated_validation_t final_validation = {
       SV_AUTH_RESULT_OK, false, 15, 14, 1, SV_PUBKEY_VALIDATION_NOT_FEASIBLE, true, 0, 0};
-  // One pending NAL Unit per GOP.
+
```

Comment:
I don't remember why we skip this test if auth_level is not FRAME (line 2204).
Shouldn't it work for both cases now?